libs/sdk-ui-filters/src/AttributeFilter@next/types.ts

```diff
@@ -124,6 +124,17 @@ export interface IAttributeFilterCoreProps {
      * @remarks
      * The meaning of the items is determined by the way the filter is specified: if the filter uses URIs,
      * then these are also interpreted as URIs, analogously with values.
+     *
+     * If using `hiddenElements`, make sure your input filter excludes the hidden elements, otherwise it could lead to
+     * non-intuitive behavior. For example, to create a filter that has some items hidden but is set to All:
+     *
+     * ```tsx
+     * const hiddenUris: ["uri1", "uri2"];
+     * // use the uris both in the filter...
+     * const filter = newNegativeAttributeFilter("displayForm", { uris: hiddenUris });
+     * // ...and in the prop
+     * return <AttributeFilter filter={filter} hiddenElements={hiddenUris} />
```

Comment:
This can be misleading - someone can understand it as an example how to do it correctly. Maybe make it more explicit... (eg 1 correct example and 1 wrong example, with proper annotations what is the correct way)